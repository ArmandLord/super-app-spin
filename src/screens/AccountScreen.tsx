import React from 'react';
import { View, StyleSheet, Image, SafeAreaView } from 'react-native';
import Text from '../components/Text/Text';
import iconLogout from '../assets/logout.png';
import iconAccount from '../assets/svg/account.png';

const AccountScreen = () => {
  return (
    <SafeAreaView style={styles.safeArea}>
      <View style={styles.container}>
        <Text style={styles.title}>Cuenta</Text>
        <View style={styles.usernameContainer}>
          <Text style={styles.username}>Maria Florencia</Text>
          <Image source={iconAccount} style={styles.accountIcon} />
        </View>
        <View style={styles.pointsContainer}>
          <Text style={styles.pointsText}>1,200 puntos</Text>
        </View>
        <Text style={styles.otherActions}>Otras acciones</Text>
        <View style={styles.logoutContainer}>
          <Image source={iconLogout} style={styles.logoutIcon} />
          <Text style={styles.logoutText}>Cierra sesión</Text>
        </View>
        <View style={styles.versionContainer}>
          <Text style={styles.version}>Versión: 1.203.430.983</Text>
        </View>
      </View>
    </SafeAreaView>
  );
};

const styles = StyleSheet.create({
  safeArea: {
    flex: 1,
    backgroundColor: '#fff', // Puedes ajustar el color de fondo aquí
  },
  container: {
    flex: 1,
    paddingHorizontal: 20,
    paddingTop: 20,
    paddingBottom: 20,
  },
  title: {
    fontSize: 18,
    fontWeight: '500',
    marginBottom: 10,
  },
  usernameContainer: {
    flexDirection: 'row', 
    alignItems: 'center',
    marginBottom: 5,
  },
  username: {
    fontSize: 24,
    marginRight: 10,
    fontWeight: 'bold',
  },
  accountIcon: {
    width: 120,
    height: 120,
  },
  pointsContainer: {
    flexDirection: 'row', 
    paddingHorizontal: 12,
    paddingVertical: 4,
    borderRadius: 12,
    backgroundColor: '#C9E9F3',
    marginBottom: 50,
  },
  pointsText: {
    fontSize: 16,
    color: '#006686',
    fontWeight: '700',
  },
  otherActions: {
    fontSize: 18,
    fontWeight: '600',
    marginBottom: 25,
    color: '#05053D',
  },
  logoutContainer: {
    flexDirection: 'row', 
    alignItems: 'center',
    marginBottom: 20,
  },
  logoutIcon: {
    width: 24,
    height: 24,
    marginRight: 20,
  },
  logoutText: {
    fontSize: 16,
    color: '#05053D',
  },
  versionContainer: {
    flex: 1,
    justifyContent: 'flex-end',
  },
  version: {
    fontSize: 12,
    color: '#B4B4C5',
    textAlign: 'center',
  },
});

export default AccountScreen;
