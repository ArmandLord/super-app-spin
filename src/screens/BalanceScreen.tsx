import React, { useRef } from 'react';
import { View, Text, Image, SafeAreaView, ScrollView, StyleSheet } from 'react-native';
import Disclaimer from '../components/Disclaimer/Disclaimer';
import AlertIcon from "../assets/Alert.png";
import TextInput from '../components/atoms/TextInput';
import InfoIcon from "../assets/info.png";
import BaseChip from '../components/atoms/Chip/BaseChip';
import PointsIcon from '../assets/svg/icon.png';
import Button from '../components/Button/Button';
import CustomCard from '../components/Card/components/CustomCard';

import { useNavigation } from '@react-navigation/native';
import PointCounter from '../components/Card/components/PointCounter';

const BalanceScreen = () => {
    const { navigate } = useNavigation();
    const points = 9999;
    const formattedPoints = points.toLocaleString();
    const pointsValue = (points / 10).toFixed(2);
    const numericInputRef = useRef(null);
    const handleInputChange = (text) => {
        console.log('Texto ingresado:', text);
    };

    return (
        <SafeAreaView style={styles.safeArea}>
          <ScrollView contentContainerStyle={styles.scrollViewContent}>
            <View style={styles.container}>
              <View style={styles.pointsContainer}>
                <View style={styles.pointsTextContainer}>
                  <Text style={styles.pointsText}>{formattedPoints} puntos</Text>
                  <Image
                    source={InfoIcon}
                    style={[styles.infoIcon, styles.infoIconMargin]}
                  />
                </View>
              </View>
              <PointCounter value={pointsValue} />
              <View style={styles.divider} />
              <Text style={styles.pointsInputLabel}>
                Escribe el valor de los puntos que quieres cambiar
              </Text>
              {points > 1000 ? (
                <View>
                  <View style={styles.cardsContainer}>
                    {points > 1000 && points <= 10000 && (
                      <View style={styles.cardRow}>
                        <CustomCard title="$50" value="500" />
                        <CustomCard title="$100" value="1000" />
                      </View>
                    )}
                    {points > 10000 && (
                      <View style={styles.cardRow}>
                        <View style={styles.cardColumn}>
                          <CustomCard title="$50" value="500" />
                          <CustomCard title="$200" value="2000" />
                        </View>
                        <View style={styles.cardColumn}>
                          <CustomCard title="$100" value="1000" />
                          <CustomCard title="$500" value="5000" />
                        </View>
                      </View>
                    )}
                  </View>
                  <Text style={styles.otherInput}>Otro:</Text>
                  <TextInput
                    ref={numericInputRef}
                    variant="numeric"
                    onChangeText={handleInputChange}
                    placeholder="Monto en pesos"
                  />
                  {points > 10000 && (
                    <Text style={styles.maxValueText}>
                      El valor máximo que puedes cambiar es $1,000.00
                    </Text>
                  )}
                </View>
              ) : (
                <TextInput
                  ref={numericInputRef}
                  variant="numeric"
                  onChangeText={handleInputChange}
                  placeholder="Monto en pesos"
                />
              )}
              {points < 200 && (
                <View style={styles.disclaimerContainer}>
                  <Disclaimer
                    text="Recuerda que necesitas tener mínimo $20.00 en puntos para poder cambiarlos con la marca que elegiste"
                    icon={AlertIcon}
                    iconColor="#955000"
                    backgroundColor="#FFDFBC"
                    textColor="#05053D"
                  />
                </View>
              )}
            </View>
          </ScrollView>
          <View style={styles.buttonContainer}>
            <Button
              text="Continuar"
              variant="primary"
              onPress={() => navigate('TicketScreen')}
            />
          </View>
        </SafeAreaView>
      );
};

const styles = StyleSheet.create({
  safeArea: {
    flex: 1,
    backgroundColor: '#fff',
  },
  scrollViewContent: {
    flexGrow: 1,
    justifyContent: 'space-between',
  },
  container: {
    flex: 1,
    padding: 20,
  },
  pointsContainer: {
    flexDirection: 'column',
    alignItems: 'flex-start',
    marginBottom: 10,
  },
  pointsTextContainer: {
    flexDirection: 'row',
    alignItems: 'center',
    marginRight: 5,
    marginBottom: 8,
  },
  pointsText: {
    fontSize: 24,
    fontFamily: 'Poppins',
    fontWeight: 'bold',
  },
  infoIcon: {
    tintColor: '#1723D3',
    width: 24,
    height: 24,
  },
  infoIconMargin: {
    marginLeft: 12,
  },
  divider: {
    height: 1,
    backgroundColor: '#E1E1E1',
    marginVertical: 18,
  },
  minimumValueText: {
    color: '#69698B',
    fontSize: 12,
    fontFamily: 'Poppins',
    marginLeft: 12,
  },
  pointsInputLabel: {
    fontSize: 18,
    fontFamily: 'Poppins',
    marginBottom: 10,
    textAlign: 'left',
  },
  minValueContainer: {
    marginTop: 10,
    flexDirection: 'row',
    alignItems: 'center',
  },
  disclaimerContainer: {
    marginTop: 10,
  },
  bottomText: {
    marginTop: 20,
  },
  cardsContainer: {
    marginVertical: 20,
  },
  cardColumn: {},
  otherInput: {
    fontSize: 18,
    fontFamily: 'Poppins',
    marginBottom: 18,
  },
  cardRow: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    marginBottom: 20,
  },
  otherInputAbove: {
    fontSize: 16,
    fontFamily: 'Poppins',
    marginBottom: 6,
  },
  buttonContainer: {
    marginBottom: 20,
    marginHorizontal: 20,
  },
  maxValueText: {
    color: 'red',
    fontSize: 12,
    fontFamily: 'Poppins',
    marginLeft: 12,
  },
});

export default BalanceScreen;
