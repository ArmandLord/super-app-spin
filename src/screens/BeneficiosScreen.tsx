import React from "react";
import { SafeAreaView, StyleSheet, View, Image} from "react-native";
import { ScrollView } from "react-native-gesture-handler";
import Text from "../components/Text/Text";
import Tag from "../components/atoms/Tag";
import StackedCardGrid from "../components/GridView/StackedCardGrid";
import BannerCarousel from "../components/molecules/BannerCarousel";

interface Banner {
    id: number,
    title: string,
    image: object,
    onPress: () => void,
}

const BeneficiosScreen = () => {
    const changePoints = require('../assets/change-points.png');
    const checkHistory = require('../assets/check-history.png');

    const handleCardPress = (title: string) => {
        console.log('Se hizo clic en:', title);
      };

      const banners: Banner[] = [
        {
          id: 1,
          title: 'Volaris',
          image: {
            uri: 'https://firebasestorage.googleapis.com/v0/b/femsa-af419.appspot.com/o/banner1.png?alt=media&token=ba2b6df3-1174-4618-b8a3-5f85e96038b6',
          },
          onPress: () => console.log('Banner 1'),
        },
        {
          id: 2,
          title: 'Andati',
          image: {
            uri: 'https://banners-tae.s3.amazonaws.com/spinplus-home-banner1-andatti.png',
          },
          onPress: () => console.log('Banner 2'),
        },
        {
          id: 3,
          title: 'Volaris',
          image: {
            uri: 'https://firebasestorage.googleapis.com/v0/b/femsa-af419.appspot.com/o/banner1.png?alt=media&token=ba2b6df3-1174-4618-b8a3-5f85e96038b6',
          },
          onPress: () => console.log('Banner 3'),
        },
      ];

    return(
        <SafeAreaView style={benefitsStyles.mainContainer}>
            <ScrollView style={benefitsStyles.mainScroll} showsVerticalScrollIndicator= {false}>
                <Text variant='title-two-semibold' style={benefitsStyles.title}>Beneficios</Text>
                <View style={benefitsStyles.container}>
                    <View style={benefitsStyles.leftContent}>
                        <Text variant='subtitle-semibold' style={benefitsStyles.subtitle}>Tienes</Text>
                        <Text variant='jumbo-two' style={benefitsStyles.points}>10,657 puntos</Text>{/*Cambiar por context*/}
                        <View>
                           <Tag text="Valen $156.00" variant='points' leftIcon={require('../assets/points.png')} size='large'/> 
                        </View>
                    </View>
                    <View style={benefitsStyles.rightContent}>
                        <Image
                        source={require('../assets/Premia1.png')}
                        style={benefitsStyles.image}
                        resizeMode="contain"
                        />
                    </View>
                </View>
                <View style={{}}>
                    <StackedCardGrid
                        data={[
                              {
                                title: 'Consulta tu historial',
                                icon: <Image source={checkHistory} style={benefitsStyles.imageGrid}/>,
                                onPress: () => handleCardPress('Consulta tu historial'),
                              },
                              {
                                title: 'Cambia tus puntos',
                                icon: <Image source={changePoints} style={benefitsStyles.imageGrid}/>,
                                onPress: () => handleCardPress('Cambia tus puntos'),
                                
                              },
                        ]}
                        titlesSize='small'
                        numberOfColumns={2}
                        containerStyle={{marginTop: 25, gap:15,}}
                    />
                </View>
                <View>
                    <Text variant='subtitle-semibold' style={benefitsStyles.subtitleSection}>Acumula productos</Text>
                    <Text variant='default-body'>Muy pronto podrás sumar tus compras y ganar productos de regalo</Text>
                    <Image 
                        source={require('../assets/acumula.png')}
                        style={benefitsStyles.imageSection}
                    />
                </View>
                <View>
                    <Text variant='subtitle-semibold' style={benefitsStyles.subtitleSection}>Gana más puntos</Text>
                    <Text variant='default-body'>Muy pronto podrás ganar más puntos en el total de tu compra</Text>
                    <Image 
                        source={require('../assets/ganamas.png')}
                        style={benefitsStyles.imageSection}
                    />
                </View>
                <View>
                    <Text variant='subtitle-semibold' style={benefitsStyles.subtitleSection}>Suma al comprar</Text>
                    <Text variant='default-body'>Muy pronto podrás acumular compras y llevarte productos de regalo</Text>
                    <Image 
                        source={require('../assets/suma.png')}
                        style={benefitsStyles.imageSection}
                    />
                </View>
                <View style={benefitsStyles.bannerCarrousel}>
                    <Text variant='subtitle-semibold' style={benefitsStyles.bannersSubtitle}>Más promociones</Text>
                    <BannerCarousel
                        variant="default"
                        banners={banners}
                        withStepper
                        stepperPosition="left"
                        bannerWidth={'80%'}
                    />
                </View>
            </ScrollView>
        </SafeAreaView>
    )
}

const benefitsStyles = StyleSheet.create({
    mainContainer: {
      flex: 1,
      backgroundColor: 'white'
    },
    mainScroll: {
        marginHorizontal: 15,
    },
    container: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'space-between',
        paddingVertical: 10,
      },
      title: {
        fontSize: 22,
        fontWeight: '500',
        marginBottom: 20,
      },
      leftContent: {
        flex: 0.65,
        marginRight: 5,
      },
      rightContent: {
        flex: 0.35,
        alignItems: 'flex-end',
      },
      subtitle: {
        fontWeight: '500',
        fontSize: 18
      },
      points: {
        marginBottom: 10,
        fontWeight: '800'
      },
      tagContent: {
        fontWeight: '600',
      },
      image: {
        width: 100,
        height: 100,
      },
      imageGrid:{
        width: 70,
        height: 70,
        resizeMode: 'cover',
        marginTop: 15,
      },
      subtitleSection:{
        fontWeight: '500',
        fontSize: 20,
        marginTop: 25,
      },
      imageSection:{
        resizeMode: 'contain',
        width: 350,
        height: 250,
        marginTop: 25,
      },
      bannerCarrousel:{
        marginVertical: 25,
      },
      bannersSubtitle:{
        marginVertical: 15,
        fontWeight: '500',
        fontSize: 20,
      }
  });

export default BeneficiosScreen;
