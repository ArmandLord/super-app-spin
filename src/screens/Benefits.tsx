import { Image, ScrollView, StyleSheet, View } from "react-native"
import Text from '../components/Text/Text'
import BaseCard from "../components/Card/components/BaseCard"
import StackedCardGrid from "../components/GridView/StackedCardGrid"
import { NativeStackScreenProps } from "@react-navigation/native-stack"
import { TStackBenefits } from "../types/navigation.types"
import { FEED, TRANSACTIONS } from "../constants/screens"

type Props = {}

export const Benefits = ({ navigation }: NativeStackScreenProps<TStackBenefits, typeof FEED>) => {
    return (
        <View style={styles.container}>
            <ScrollView>
                <StackedCardGrid containerStyle={{ flex: 0.5 }} titlesSize='small' numberOfColumns={2} idForTestContainer="grid" data={[
                    {
                        title: 'Consulta tu historial',
                        icon: <Image source={require('../assets/Images/star.png')} />,
                        onPress: () => navigation.navigate(TRANSACTIONS)
                    },
                    {
                        title: 'Cambia tus puntos',
                        icon: <Image source={require('../assets/Images/ticket.png')} />,
                        onPress: () => console.log("la 2")
                    },
                ]} />
                <BaseCard>
                    <Text testID="promo-1-text" variant="default-body-bold" numberOfLines={2}>
                        Acumula productos
                    </Text>
                    <Text variant="default-body-bold" numberOfLines={2}>
                        Muy pronto...
                    </Text>
                    <Image source={require('../assets/Images/star-card.png')} style={{ height: 50, width: 50 }} />
                </BaseCard>
                <BaseCard>
                    <Text testID="promo-2-text" variant="default-body-bold" numberOfLines={2}>
                        Gana m√°s puntos
                    </Text>
                    <Text variant="default-body-bold" numberOfLines={2}>
                        Muy pronto...
                    </Text>
                    <Image source={require('../assets/Images/star-plus.png')} style={{ height: 50, width: 50 }} />
                </BaseCard>
                <BaseCard>
                    <Text testID="promo-3-text" variant="default-body-bold" numberOfLines={2}>
                        Suma al comprar
                    </Text>
                    <Text variant="default-body-bold" numberOfLines={2}>
                        Muy pronto...
                    </Text>
                    <Image source={require('../assets/Images/star-ribbon.png')} style={{ height: 50, width: 50 }} />
                </BaseCard>
            </ScrollView>
        </View>
    )
}

const styles = StyleSheet.create({
    container: { flex: 1 }
})