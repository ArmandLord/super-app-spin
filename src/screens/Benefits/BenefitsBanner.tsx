import { Image, ScrollView } from 'react-native';
import styles from '../../styles/spinplus/Benefits/BenefitsBanner.styles';

const data = [
  require('../../assets/banner.png'),
  require('../../assets/banner.png'),
  require('../../assets/banner.png'),
  require('../../assets/banner.png'),
];

export const BenefitsBanner = () => (
  <ScrollView
    horizontal
    style={styles.container}
    contentContainerStyle={styles.banner}
    showsHorizontalScrollIndicator={false}>
    {data.map((image, index) => (
      <Image source={image} key={`img-${index}`} />
    ))}
  </ScrollView>
);
