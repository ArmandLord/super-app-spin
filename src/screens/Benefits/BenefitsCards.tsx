import { View, Image, useWindowDimensions, ImageProps } from 'react-native';
import { TouchableOpacity } from 'react-native-gesture-handler';
import Card from '../../components/Card/Card';
import Text from '../../components/Text/Text';
import styles from '../../styles/spinplus/Benefits/BenefitsCards';

type BenefitCardT = {
  text: string;
  width: number;
  img: ImageProps;
  // REPLACE FOR SCREENS TYPE
  navigateTo: string;
};

// TODO: Add correct navigation on onPress action of the card when Movimientos and Cambio de puntos screen is ready
const BenefitCard = ({ img, text, navigateTo, width }: BenefitCardT) => {
  // ADD NAVIGATION HOOK WHEN MERGED NAVIGATION
  return (
    <View style={{ width }}>
      <Card style={styles.cardContainer} testID="benefit-card">
        <TouchableOpacity>
          <Image style={styles.img} width={124} height={124} source={img} />
          <View style={styles.cardTextContainer}>
            <Text variant="small-body-bold" style={styles.cardText}>
              {text}
            </Text>
          </View>
        </TouchableOpacity>
      </Card>
    </View>
  );
};

export const BenefitsCards = () => {
  const { width } = useWindowDimensions();
  const twoColumnsCardWidth = width / 2 - 24;

  return (
    <View style={styles.container} testID="benefit-cards">
      <BenefitCard
        navigateTo=""
        width={twoColumnsCardWidth}
        text={`Consulta \n tu historial`}
        img={require('../../assets/benefits-history.png')}
      />
      <BenefitCard
        navigateTo=""
        width={twoColumnsCardWidth}
        text="Cambia tus puntos"
        img={require('../../assets/change-points.png')}
      />
    </View>
  );
};
