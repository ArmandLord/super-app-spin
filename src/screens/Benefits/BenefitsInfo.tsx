import { View, Image } from 'react-native';
import Text from '../../components/Text/Text';
import styles from '../../styles/spinplus/Benefits/BenefitsInfo.styles';

// TODO: Replace hardcoded text with retrieved from endpoint
export const BenefitsInfo = () => {
  return (
    <View style={styles.container} testID="benefits-info">
      <View>
        <Text style={styles.pointsText} variant="headline-large">
          10, 657 puntos
        </Text>
        <View style={styles.pointsContainer}>
          <Image
            width={16}
            height={16}
            source={require('../../assets/starburst.png')}
          />
          <Text style={styles.pointsValueText} variant="label-extra-small-bold">
            Valen $156.00
          </Text>
        </View>
      </View>
      <Image
        width={120}
        height={120}
        source={require('../../assets/spinPremiaRibbon.png')}
      />
    </View>
  );
};
