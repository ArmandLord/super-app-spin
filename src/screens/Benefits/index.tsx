import { ScrollView } from 'react-native';
import { BenefitsInfo } from './BenefitsInfo';
import { BenefitsCards } from './BenefitsCards';
import { BenefitsSection } from './BenefitsSection';
import styles from '../../styles/spinplus/Benefits/Benefits.styles';

export const Benefits = () => {
  return (
    <ScrollView
      style={styles.container}
      contentContainerStyle={styles.contentContainer}>
      <BenefitsInfo />
      <BenefitsCards />
      <BenefitsSection
        title="Acumula produtos"
        img={require('../../assets/acumulaProductos.png')}
        text="Muy pronto podrás sumar tus compras y ganar productos de regalo"
      />
      <BenefitsSection
        title="Gana más puntos"
        img={require('../../assets/ganaPuntos.png')}
        text="Muy pronto podrás ganar más puntos en el total de tu compra"
      />
      <BenefitsSection
        title="Suma al comprar"
        img={require('../../assets/sumaAlComprar.png')}
        text="Muy pronto podrás acumular compras y llevarte productos de regalo"
      />
    </ScrollView>
  );
};
