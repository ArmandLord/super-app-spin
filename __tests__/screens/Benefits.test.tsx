import React from 'react';
import { render } from '@testing-library/react-native';
import { Benefits } from '../../src/screens/Benefits';
import { BenefitsInfo } from '../../src/screens/Benefits/BenefitsInfo';
import { BenefitsCards } from '../../src/screens/Benefits/BenefitsCards';
import { BenefitsBanner } from '../../src/screens/Benefits/BenefitsBanner';

describe('Benefits', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<Benefits />);
    expect(toJSON()).toMatchSnapshot();
  });

  describe('Benefits Info', () => {
    it('renders correctly', () => {
      const { getByTestId } = render(<BenefitsInfo />);
      const benefitsInfo = getByTestId('benefits-info');

      expect(benefitsInfo).toBeDefined();
      expect(benefitsInfo).toHaveLength(2);
    });
  });

  describe('Benefits cards', () => {
    it('renders correctly', () => {
      const { getByTestId } = render(<BenefitsCards />);
      const benefitsCards = getByTestId('benefits-cards');

      expect(benefitsCards).toBeDefined();
      expect(benefitsCards).toHaveLength(2);
    });
  });

  describe('Benefits Banner', () => {
    it('renders correctly', () => {
      const { toJSON } = render(<BenefitsBanner />);
      expect(toJSON()).toMatchSnapshot();
    });
  });
});
