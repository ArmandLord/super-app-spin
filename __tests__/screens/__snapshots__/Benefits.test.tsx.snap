// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefits Benefits Banner renders correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    {
      "gap": 16,
      "paddingVertical": 24,
    }
  }
  horizontal={true}
  showsHorizontalScrollIndicator={false}
  style={
    {
      "flexGrow": 0,
    }
  }
>
  <View>
    <Image
      source={
        {
          "testUri": "../../../src/assets/banner.png",
        }
      }
    />
    <Image
      source={
        {
          "testUri": "../../../src/assets/banner.png",
        }
      }
    />
    <Image
      source={
        {
          "testUri": "../../../src/assets/banner.png",
        }
      }
    />
    <Image
      source={
        {
          "testUri": "../../../src/assets/banner.png",
        }
      }
    />
  </View>
</RCTScrollView>
`;
